src/plugins/intel_gpu/src/kernel_selector/common/common_types.h

```diff
@@ -181,6 +181,15 @@ enum class PoolType {
     DEFORMABLE_BILINEAR
 };
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// AlignedType
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+enum class RoiAlignedType {
```

Comment:
why it's needed in common_types? Cannot it be defined near parameters?

Reply:
Done